File: app/src/main/java/kr/co/tjoeun/colosseum_kotlin/BaseActivity.java
package kr.co.tjoeun.colosseum_kotlin;

import android.content.Context;

import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;

public abstract class BaseActivity extends AppCompatActivity {

    public Context mContext = this;

    public abstract void setupEvents();
    public abstract void setValues();

    public void setCustomActionBar(){
        getSupportActionBar().setDisplayOptions(ActionBar.DISPLAY_SHOW_CUSTOM);
        getSupportActionBar().setCustomView(R.layout.custion);
    }

}

File: app/src/main/java/kr/co/tjoeun/colosseum_kotlin/BaseActivity.kt
package kr.co.tjoeun.colosseum_kotlin

import android.os.Bundle
import android.text.Html
import androidx.appcompat.app.AppCompatActivity

abstract class BaseActivity : AppCompatActivity() {

    val mContext = this

    abstract fun setupEvents()
    abstract fun setValues()

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setTitle(Html.fromHtml("<font color='#000000'>제목변경</font>"))
    }

}